src/code42cli/cmds/cases.py

```diff
@@ -0,0 +1,241 @@
+import click
+from py42.clients.cases import CaseStatus
+from py42.exceptions import Py42BadRequestError
+from py42.exceptions import Py42NotFoundError
+
+from code42cli.click_ext.groups import OrderedGroup
+from code42cli.options import format_option
+from code42cli.options import sdk_options
+from code42cli.output_formats import OutputFormatter
+
+
+case_number_arg = click.argument("case-number", type=int)
+name_option = click.option("--name", help="Name of the case.",)
+assignee_option = click.option("--assignee", help="User UID of the assignee.")
+description_option = click.option("--description", help="Description of the case.")
+notes_option = click.option("--notes", help="Notes on the case.")
+subject_option = click.option("--subject", help="User UID of a subject of the case.")
+status_option = click.option(
+    "--status",
+    help="Status of the case. `OPEN` or `CLOSED`",
+    type=click.Choice(CaseStatus.choices()),
+)
+file_event_id_option = click.option(
+    "--event-id", required=True, help="File event id associated to the case."
+)
+
+
+def _get_cases_header():
+    return {
+        "number": "number",
```

Comment:
Can we add case number here?

I thought I made this comment already but can't find it.

Reply:
Number above refers to `case number`, do you want me to rename it?

Reply:
No I made this comment before doing re-checking out the branch after your last force-commit